src/main/java/taxi/controller/LoginController.java

```diff
@@ -0,0 +1,42 @@
+package taxi.controller;
+
+import java.io.IOException;
+import javax.servlet.ServletException;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import javax.servlet.http.HttpSession;
+import taxi.exception.AuthenticationException;
+import taxi.lib.Injector;
+import taxi.model.Driver;
+import taxi.service.AuthenticationService;
+import taxi.service.AuthenticationServiceImpl;
+
+public class LoginController extends HttpServlet {
+    private static final Injector injector
+            = Injector.getInstance("taxi");
+    private final AuthenticationServiceImpl authenticationService
+            = (AuthenticationServiceImpl) injector.getInstance(AuthenticationService.class);
```

Comment:
Use AuthenticationService interface instead of AuthenticationServiceImpl for the object's type.